### PR TITLE
OCP 4.16 nightly CI for oadp-operator 1.4 branch

### PR DESCRIPTION
**Pull request by mpryc**
Introduction of OCP 4.16 jobs for oadp-operator 1.4 branch.

### CODE REVIEW

**Comment by mpryc** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws

**Comment by mpryc** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-gcp

**Comment by mpryc** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws

**Comment by mpryc** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws

File: ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.16.yaml
Comment (mateusoliveira43) [inline ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.4__4.16.yaml:1]: I would also add this job to master

With Scott's PR getting merged, I believe we will start to do normal work (master then cherry-pick to release branch). If we add just to oadp-1.4, there is the possibility that a cherry-pick fails on release branch, but we did not see the error on master

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws

**Comment by mateusoliveira43** [comment]
/pj-rehearse pull-ci-openshift-oadp-operator-oadp-1.4-4.16-e2e-test-kubevirt-aws

**Review by weshayutin — APPROVED** [review]
/LGTM

**Comment by mpryc** [comment]
This PR did ran aws previously with success, not triggering again as the last commit was to remove gcp and azure

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**